tests/neuro_san/client/agent_evaluator_factory.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (C) 2023-2025 Cognizant Digital Business, Evolutionary AI.
+# All Rights Reserved.
+# Issued under the Academic Public License.
+#
+# You can be released from the terms, and requirements of the Academic Public
+# License by purchasing a commercial license.
+# Purchase of a commercial license is mandatory for any use of the
+# neuro-san SDK Software in commercial settings.
+#
+# END COPYRIGHT
+
+from tests.neuro_san.client.agent_evaluator import AgentEvaluator
+from tests.neuro_san.client.assert_forwarder import AssertForwarder
+from tests.neuro_san.client.keywords_agent_evaluator import KeywordsAgentEvaluator
+
+
+class AgentEvaluatorFactory:
+    """
+    Factory that creates AgentEvaluators
+    """
+
+    @staticmethod
+    def create_evaluator(asserts: AssertForwarder, evaluation_type: str, test_key: str) -> AgentEvaluator:
```

Comment:
Given a test entry for a response, this factory creates an evaluator which corresponds to keys found in the response block of an interaction.